contrib/labstack/echo.v4/option.go

```diff
@@ -18,6 +18,7 @@ type config struct {
 	analyticsRate     float64
 	noDebugStack      bool
 	ignoreRequestFunc IgnoreRequestFunc
+	ignoreError       IgnoreErrorFunc
```

Comment:
```suggestion
	errCheck      func(err error) bool
```

(to stay consistent with other contribs)